packages/components/src/components/tooltip/tooltip.test.ts

```diff
@@ -7,7 +7,7 @@ describe('<sd-tooltip>', () => {
   it('should be visible with the open attribute', async () => {
     const el = await fixture<SdTooltip>(html`
       <sd-tooltip content="This is a tooltip" open>
-        <sd-button>Hover Me</sd-button>
+        <sd-icon>Click Me</sd-icon>
```

Comment:
I know this is only for testing but why didn't you add an icon name here? even if its not needed, someone could see a technically wrong usage of our component... Or do you think its too much code as its not used anyways?

Or did I miss anything in the logic of the component?

Reply:
Hmm I see your point, it felt like a bit too much code that's not needed, however with the new changes it will stay as a button after all.